packages/patternfly-4/react-docs/src/components/navigation/navigation.js

```diff
@@ -58,10 +58,13 @@ class Navigation extends React.Component {
             </Link>
           </div>
           <div className={css(styles.search)}>
```

Comment:
I dunno if it will be div-itis to do this, but you could change this to

```css
<div class="pf-c-form">
    <div class="pf-c-form__group">
        <label class="pf-c-form__label">
            <span class="pf-c-title pf-m-lg"> /* or use a different modifier for a different size*/
        <input class="pf-c-form-control">
```

That would align with https://pf-next.com/components/form/examples/